### PR TITLE
I&D, CDK: remove polygon 2.0

### PR DESCRIPTION
**Pull request by hsutaiyu**
This PR covers:

- Remove references to Polygon 2.0 to avoid confusion.
- Change "Interop layer" -> "Agglayer"
- Add EIP links
- Miden testnet update
- Other minor fixes

### CODE REVIEW

File: docs/cdk/glossary/index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The native bridge infrastructure of CDK chains that allows transfer of assets an
 A contract on the LxLy bridge utilizes its message passing capabilities to pass arbitrary messages between L1 and L2.  This is not a separate component, and is part of the LxLy bridge's architecture.
```
Comment (EmpieichO) [inline docs/cdk/glossary/index.md]:
```suggestion
A contract on the LxLy bridge utilizes its message passing capabilities to pass arbitrary messages between L1 and L2.  This is not a separate component, but part of the LxLy bridge's architecture.
```

File: docs/innovation-design/polygon-protocols.md [hunk 2]
```diff
@@ -56,10 +59,10 @@ Upcoming release: The next release of Polygon CDK, expected in Q1 2024, will sup
 
 ## Polygon Miden
 
-Polygon Miden is an in-development, novel ZK rollup designed as a zkVM. Powered by the Miden VM, Polygon Miden extends Ethereum’s feature set, providing throughput and privacy features that are beyond the capabilities of the EVM. 
+Polygon Miden is a novel ZK rollup currently in development, designed as a zkVM. Powered by the Miden VM, Polygon Miden extends Ethereum’s capabilities, offering enhanced throughput and privacy features beyond those of the EVM.
 
 ### Where Polygon Miden is heading
 
 A testnet for Polygon Miden is expected to launch in Q1 2024, with mainnet following later in the year.
```
Comment (EmpieichO) [inline docs/innovation-design/polygon-protocols.md]: Miden testnet was launched some weeks ago. Find the link (search Slack messages by Gobinath on Miden) ... to the testnet, and rephrase as ...
"[Polygon Miden testnet](... insert link here ...) was launched in Q1 2024, with mainnet is expected to follow later in the year."

File: docs/innovation-design/welcome.md [hunk 3]
```diff
@@ -4,21 +4,21 @@ comments: true
...
 
-## Polygon 2.0: The basics
+## The basics
 
-Polygon technologies will help developers build in an elastically scalable and unified ecosystem of ZK-powered Layer 2s on Ethereum, where users can create, program and exchange value.
+Polygon technologies will help developers build in an elastically scalable and *unified* ecosystem of ZK-powered Layer 2s on Ethereum, where users can create, program and exchange value.
 
-The Polygon 2.0 vision is a unified multichain ecosystem. A web of interoperable ZK-powered Ethereum L2s, with near-instant and atomic L2 <> L2 transactions, and designed to empower developers to build without limitations. Developers will choose to build dApps, design and launch dedicated application-specific L2 chains, or migrate existing EVM Layer 1 chains to become an L2.
+Polygon envisions a unified multichain ecosystem. A web of interoperable ZK-powered Ethereum L2s, with near-instant and atomic L2 <-> L2 transactions, and designed to empower developers to build without limitations. Developers will choose to build dApps, design and launch dedicated application-specific L2 chains, or migrate existing EVM Layer 1 chains to become an L2.
```
Comment (EmpieichO) [inline docs/innovation-design/welcome.md]:
```suggestion
Polygon envisions a unified multichain ecosystem. A web of interoperable ZK-powered Ethereum L2s, with near-instant and atomic L2 <-> L2 transactions, designed to empower developers build without limitations. An ecosystem where developers can choose to build dApps, design and launch dedicated application-specific L2 chains, or migrate existing EVM Layer 1 chains to become L2s.
```

File: docs/innovation-design/welcome.md [hunk 4]
```diff
@@ -4,21 +4,21 @@ comments: true
...
 
-The endgame of Polygon 2.0 is for developers to build in an environment that feels and functions more like the internet. This means a blockchain ecosystem that can scale without limit, seamlessly unified, and backed by the decentralization and security of Ethereum.
+The endgame is to allow developers to build in an environment that feels and functions more like the internet. This means a blockchain ecosystem that can scale without limit, seamlessly unified, and backed by the decentralization and security of Ethereum.
```
Comment (EmpieichO) [inline docs/innovation-design/welcome.md:15]:
```suggestion
The endgame is to allow developers build in an environment that feels and functions more like the internet. This means a blockchain ecosystem that can scale without limit, seamlessly unified, and backed by the decentralization and security of Ethereum.
```

File: docs/innovation-design/welcome.md [hunk 5]
```diff
@@ -4,21 +4,21 @@ comments: true
...
 
-The best way to accomplish this goal is through zero-knowledge cryptography as it is capable of providing verifiable proofs that attest to the integrity of off-chain computations. Otherwise, scaling technologies often have to add additional social-economic mechanisms to mediate off-chain computations. The consequence of which is delayed settlement of transactions.
+The best way to accomplish this goal is through zero-knowledge cryptography as it is capable of providing verifiable proofs that attest to the integrity of off-chain computations. Otherwise, scaling technologies often have to add additional socio-economic mechanisms to mediate off-chain computations, which often results in delayed settlement of transactions.
 
-Polygon 2.0 applies the open source, zero-knowledge scaling technology developed at Polygon Labs, and this will allow Ethereum to scale to the limits of the internet.
+Future iterations will focus on applying the open source, zero-knowledge scaling technology developed at Polygon Labs, and this will allow Ethereum to scale to the limits of the internet.
```
Comment (EmpieichO) [inline docs/innovation-design/welcome.md]:
```suggestion
Future iterations will focus on applying the open source, zero-knowledge scaling technology developed at Polygon Labs, and allow Ethereum to scale to the limits of the internet.
```

**Review by EmpieichO — APPROVED** [review]

File: docs/innovation-design/welcome.md [hunk 6]
```diff
@@ -4,21 +4,21 @@ comments: true
...
```
Comment (hsutaiyu) [inline docs/innovation-design/welcome.md]:
```suggestion
Future iterations will focus on applying the open source, zero-knowledge scaling technology developed at Polygon Labs, and these advancements will enable Ethereum to scale to the limits of the internet..
```

File: docs/innovation-design/polygon-protocols.md (hunk 2 above)
Comment (hsutaiyu) [reply docs/innovation-design/polygon-protocols.md]: Good point. Updated with a new commit.

**Pull request merged**